src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -37,6 +37,7 @@ public void Dispose()
         }
 
         [Theory]
+        [ActiveIssue(8437)]
```

Comment:
This can at least be scoped to Windows, even if we don't have a good way to scope it to Windows 7, right?